### PR TITLE
copy update: /data/opensearch/support

### PR DESCRIPTION
**Pull request by jasmine-canonical**
## Done
- Updates /data/opensearch/support to add "multiple packaging formats" section

## QA

- View the site locally in your web browser at: https://canonical-com-1425.demos.haus/data/opensearch/support
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that /data/opensearch/support matches the [copy doc](https://docs.google.com/document/d/1ovtXv8RBybw-flvttyqWOzZivUryQMP34T1UGe90HlY/edit?usp=sharing)

## Issue / Card

Fixes [WD-16793](https://warthogs.atlassian.net/browse/WD-16793)

[WD-16793]: https://warthogs.atlassian.net/browse/WD-16793?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Sophie-32** [comment]
Body text should be at the same level as the heading like the section below, see for reference - 
Screen Shot 2024-11-14 at 5 09 58 PM

**Comment by mtruj013** [comment]
@jasmine-canonical hey, thanks for your work on this! In the future could you please include the demo link with the relevant path in your pr description? This is documented as step 7 in the [copy update guide ](https://webteam.canonical.com/practices/copy-doc-updates). I've update the description to reflect how it should look. Thanks again and lmk if you have any questions :)

**Comment by jasmine-canonical** [comment]
> Body text should be at the same level as the heading 

In the copy doc, it looks like the templates are different between the sections. This one is labelled as "TEMPLATE OPTIONS" whereas the one below is labelled "TEMPLATE BASIC TEXT". Should I still change it so that the body text is level with the heading?

**Comment by Sophie-32** [comment]
Aha! Thank you for pointing that out, the one labelled options is mislabbeled, that is also a basic text so should have theh same formatting. Thank you!

File: templates/data/opensearch/support.html [hunk 1]
```diff
@@ -194,6 +194,49 @@ <h3 class="p-heading--5">Fast response time</h3>
...
+      <hr class="p-rule" />
+      <div class="col-6">
+        <h2>Multiple packaging formats</h2>
+      </div>
+      <div class="col-6">
+        <p>
+          We provide maintained and supported OpenSearch packages in both snap and container formats:
+        </p>
+      </div>
+    </div>
```
Comment (mtruj013) [inline templates/data/opensearch/support.html:208]: I think it's best to use 50-50 split here. This makes the row responsive without needing to add additional col classes. Also, please wrap the row in the `p-section--shallow` instead, it's not recommended to mix grid and component classes like this

```suggestion
  <div class=p-section--shallow">
    <div class="row--50-50">
        <hr class="p-rule" />
        <div class="col">
          <h2>Multiple packaging formats</h2>
        </div>
        <div class="col">
          <p>
            We provide maintained and supported OpenSearch packages in both snap and container formats:
          </p>
        </div>
      </div>
  </div>
    
```

File: templates/data/opensearch/support.html [hunk 2]
```diff
@@ -194,6 +194,49 @@ <h3 class="p-heading--5">Fast response time</h3>
...
+    <div class="row">
+      <div class="col-9 col-medium-6 col-start-large-4">
+        <hr class="p-rule--muted" />
+        <div class="row p-section--shallow">
```
Comment (mtruj013) [inline templates/data/opensearch/support.html:212]: Same here, these should be decoupled

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Thanks @jasmine-canonical, a couple comments in the code:

File: templates/data/opensearch/support.html (hunk 2 above)
Comment (mtruj013) [reply templates/data/opensearch/support.html:214]: Sure thing, I meant that grid classes like `row` and components like `p-section--shallow` shouldn't be used on the same element (see discussion [here](https://chat.canonical.com/canonical/pl/yjjhnfdumtf5zm9ften6pghxeo)). It should be fixed on the lines you linked as well, good catch!

**Review by mtruj013 — APPROVED** [review]
Thanks @jasmine-canonical , LGTM!

**Pull request merged**